salt/modules/gcp_addon.py

```diff
@@ -6,16 +6,22 @@
 For each packet leaving a virtual machine, the system searches that machine's
 routing table for a single best matching route.
 
+.. versionadded:: Oxygen
+
 This module will create a route to send traffic destined to the Internet
 through your gateway instance.
 
 :codeauthor: :email:`Pratik Bandarkar <pratik.bandarkar@gmail.com>`
 :maturity:   new
 :depends:    google-api-python-client
 :platform:   Linux
+
 '''
+
+# Import Python libs
 from __future__ import absolute_import
```

Comment:
Should you add the unicode_literals here. Is this in the lint checker now.

Reply:
Thanks @damon-atkins. The lint check is added, but not enabled yet. @terminalmage and @garethgreenaway are still working on this, but the unicode_literals stuff will be completed before `oxygen` is released.